test/acceptance/tools/resources/config_template.js

```diff
@@ -25,7 +25,7 @@ config.resource = {
         /**
          * Administration port for the proxy.
          */
-        adminPort: 11211
+        adminPort: {{administration_port}}
```

Comment:
mmm... this seems unhardwiring... so probably my previous comment doesn't make sense (in a second look, it seems that in that place you are definigint the default parameter in the function signature).